ocs_ci/ocs/ui/base_ui.py

```diff
@@ -882,8 +883,14 @@ def navigate_block_pool_page(self):
 
         """
         logger.info("Navigate to block pools page")
-        self.navigate_to_ocs_operator_page()
-        self.do_click(locator=self.page_nav["block_pool_link"])
+        blockpool_page = (
+            PageNavigator()
+            .nav_odf_default_page()
+            .nav_storage_systems_tab()
+            .nav_storagecluster_storagesystem_details()
```

Comment:
```
PageNavigator()
.nav_odf_default_page()
.nav_storage_systems_tab()
.nav_storagecluster_storagesystem_details()
``` returns storage_system_details page, not blockpool_page this way you will invoke nav_ceph_blockpool from storage_system_details which is logical -> `blockpool_page.nav_ceph_blockpool()`